src/main/java/cloud/fogbow/ms/core/service/BlackList.java

```diff
@@ -0,0 +1,100 @@
+package cloud.fogbow.ms.core.service;
+
+import java.util.ArrayList;
+import java.util.List;
+
+import cloud.fogbow.common.exceptions.ConfigurationErrorException;
+import cloud.fogbow.ms.constants.ConfigurationPropertyKeys;
+import cloud.fogbow.ms.constants.Messages;
+import cloud.fogbow.ms.core.MembershipService;
+import cloud.fogbow.ms.core.PropertiesHolder;
+
+public class BlackList implements MembershipService {
+
+    private static final String SEPARATOR = ",";
+
+    private List<String> membersList;
+    private List<String> notAuthorizedTargetMembers;
+    private List<String> notAuthorizedRequesterMembers;
+    
+    public BlackList() throws ConfigurationErrorException {
```

Comment:
Let's use the politically correct term Blocklist, instead of Blacklist.

Reply:
Done.